### PR TITLE
Updated lints in easel.

### PR DESCRIPTION
**Pull request by kjawadDeveloper2**
Updated the lints on easel. 
Solved the lint issues.

### CODE REVIEW

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart [hunk 1]
```diff
@@ -134,13 +133,11 @@ class NFTsListTile extends StatelessWidget {
...
                 ),
-              )
-            : Padding(
+              ) else Padding(
                 padding: EdgeInsets.symmetric(horizontal: 20.w),
                 child: getPublishedCard(context: context),
               ),
-        (publishedNFT.assetType.toAssetTypeEnum() != AssetType.ThreeD)
-            ? IgnorePointer(
+        if (publishedNFT.assetType.toAssetTypeEnum() != AssetType.ThreeD) IgnorePointer(
```
Comment (Kamran-Khan-Gene) [inline easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:140]: IgnorePointer widget should start from new line.

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart [hunk 2]
```diff
@@ -217,8 +214,7 @@ class NFTsListTile extends StatelessWidget {
                     },
                   ),
                 ),
-              )
-            : const IgnorePointer(
+              ) else const IgnorePointer(
```
Comment (Kamran-Khan-Gene) [inline easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:217]: widget should start from new line.

**Review by Amean — APPROVED** [review]

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart (hunk 1 above)
Comment (kjawadDeveloper2) [reply easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:140]: No necessary. Thats the thing that formatter decides.

File: easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart (hunk 2 above)
Comment (kjawadDeveloper2) [reply easel/lib/screens/creator_hub/widgets/nfts_list_tile.dart:217]: No necessary. Thats the thing that formatter decides.

**Review by kamran8545 — APPROVED** [review]

**Pull request merged**